aips/aip-x.md

```diff
@@ -0,0 +1,707 @@
+| fields.issuedAt       | ISO 8601 string |           | `date-time` that indicates the issuance time.                                                                                         |
+| fields.expirationTime | ISO 8601 string |           | `date-time` that indicates when the signed authentication message is no longer valid.                                                 |
+| fields.notBefore      | ISO 8601 string |           | `date-time` that indicates when the signed authentication message starts being valid.                                                 |
+| fields.requestId      | string          |           | System-specific identifier used to uniquely refer to the authentication request.                                                      |
+| fields.resources      | string[]        |           | List of information or references to information the user wishes to have resolved as part of the authentication by the relying party. |
+| plainText             | string          | ✓         | The constructed message using the standardized message format.                                                                        |
+| signature             | bytes           | ✓         | The signature of the message. This must conform to the domain separated signing algorithm.                                            |
+| type                  | string          | ✓         | The public key scheme used to sign the message (e.g. 'ed25519', 'single_key', 'multi_key', 'multi_ed25519', etc.)                     |
+
+### AptosSignInOutput
+
+When the user has approved and signed the message, the wallet will respond with an `AptosSignInOutput` to the application. The application should then use this response to verify if the outputs are valid with the inputs.
+
+| Name              | Type          | Mandatory | Description                                                                                                                        |
+| ----------------- | ------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| account.address   | string        | ✓         | The address of the user that is signing in.                                                                                        |
+| account.publicKey | PublicKey     | ✓         | The public key of the user that is signing in.                                                                                     |
+| message           | SignInMessage | ✓         | A SignInMessage object that contains information about the message that was signed, it's signature, and plain text representation. |
+| signature         | Signature     | ✓         | The signature of the message.                                                                                                      |
```

Comment:
If we already had a signature above over (some serialization of) `AptosSignInMessage`, then what is this signature?